dcmdata/include/dcmtk/dcmdata/dcdeftag.h

```diff
@@ -5138,79 +5139,78 @@
-#define DCM_ScivisDegradationProjs               DcmTagKey(0x6001, 0x1032)
-#define DCM_ScivisSmoothVol                      DcmTagKey(0x6001, 0x1033)
-#define DCM_ScivisDegradationVol                 DcmTagKey(0x6001, 0x1034)
-#define DCM_ScivisUseResolutionRecovery          DcmTagKey(0x6001, 0x1035)
-#define DCM_ScivisNrOfSlices                     DcmTagKey(0x6001, 0x1036)
-#define DCM_ScivisWidthOfSlice                   DcmTagKey(0x6001, 0x1037)
-#define DCM_ScivisSizeOfVoxel                    DcmTagKey(0x6001, 0x1038)
-#define DCM_ScivisRecVolDiameter                 DcmTagKey(0x6001, 0x1039)
-#define DCM_ScivisRecVolLength                   DcmTagKey(0x6001, 0x103a)
-#define DCM_ScivisUsefulRecVolDiameter           DcmTagKey(0x6001, 0x103b)
-#define DCM_ScivisUsefulRecVolLength             DcmTagKey(0x6001, 0x103c)
-#define DCM_ScivisNumberOfRecoGroups             DcmTagKey(0x6001, 0x103d)
-#define DCM_ScivisRecoGroupInfoSequence          DcmTagKey(0x6001, 0x103e)
-#define DCM_ScivisNumberOfIterations             DcmTagKey(0x6001, 0x103f)
-#define DCM_ScivisSizeOfSubsets                  DcmTagKey(0x6001, 0x1040)
-#define DCM_ScivisScaleType                      DcmTagKey(0x6001, 0x1041)
-#define DCM_ScivisScaleValue                     DcmTagKey(0x6001, 0x1042)
-#define DCM_ScivisFlipStacking                   DcmTagKey(0x6001, 0x1043)
-#define DCM_ScivisFlipVertical                   DcmTagKey(0x6001, 0x1044)
-#define DCM_ScivisFlipHorizontal                 DcmTagKey(0x6001, 0x1045)
-#define DCM_ScivisUseQuantification              DcmTagKey(0x6001, 0x1046)
-#define DCM_ScivisAFlag                          DcmTagKey(0x6001, 0x1047)
-#define DCM_ScivisTFlag                          DcmTagKey(0x6001, 0x1048)
-#define DCM_ScivisHFlag                          DcmTagKey(0x6001, 0x1049)
-#define DCM_QuantificationFactor_kBq_muL         DcmTagKey(0x6001, 0x10a0)
-#define DCM_QuantificationFactor_Voxel           DcmTagKey(0x6001, 0x10a1)
-#define DCM_QuantificationFactor_status          DcmTagKey(0x6001, 0x10a2)
-#define DCM_ScalingFactor                        DcmTagKey(0x6001, 0x10a3)
-#define DCM_NuclideId                            DcmTagKey(0x6001, 0x10a4)
-#define DCM_NuclideName                          DcmTagKey(0x6001, 0x10a5)
-#define DCM_SnomedCode                           DcmTagKey(0x6001, 0x10a6)
-#define DCM_GammaIntensity                       DcmTagKey(0x6001, 0x10a7)
-#define DCM_NuclideEffectiveEnergy               DcmTagKey(0x6001, 0x10a8)
-#define DCM_Halflife_sec                         DcmTagKey(0x6001, 0x10a9)
-#define DCM_AptGroup                             DcmTagKey(0x6001, 0x10aa)
-#define DCM_AptGroupDescription                  DcmTagKey(0x6001, 0x10ab)
-#define DCM_ScivisCompressedData                 DcmTagKey(0x6001, 0x10ff)
-#define DCM_SUVScaleFactor                       DcmTagKey(0x7053, 0x0000)
-#define DCM_Unknown                              DcmTagKey(0x7053, 0x0001)
```

Comment:
Glad these DCM_Unknown tags are getting removed, I was going to ask about this since it causes a build warning.